Week 4 Reflection Contribution from mwong70-stat697

mwong70-stat660 opened the pull request:
* reopened since accidentally closed.

prof-lankham commented:
Branch name incorrect; PR being closed. Wrong file edited. PR content incomplete and inconsistently labelled. Please review instructions, open a new PR, and DM the instructor in Slack with PR URL when ready for reevaluation.

mwong70-stat660 committed:
Merge pull request #1 from stat697/master

week-4

mwong70-stat660 committed:
Merge pull request #2 from stat697/master

initiate

mochi1uv committed:
Update mwong70-stat697.md

mwong70-stat660 committed:
Merge pull request #3 from mwong70-stat697/week-4

Update mwong70-stat697.md

mwong70-stat660 committed:
Update mwong70-stat697.md

Added Answer (mwong70-stat697):

mwong70-stat660 committed:
Merge pull request #4 from stat697/master

initiate week 4 reflection resubmission

mochi1uv committed:
Update mwong70-stat697.md

Week 4 Reflection Resubmission:
- PR Formatting Specifications.
    Updated headings to the sources of the concepts being asked about.

mochi1uv committed:
Update mwong70-stat697.md

Corrected typo in L117 libref <- libredf
